packages/manager/src/components/Dialog/Dialog.tsx

```diff
@@ -95,7 +95,8 @@ export const Dialog = (props: DialogProps) => {
 };
 
 const StyledDialog = styled(_Dialog, {
-  shouldForwardProp: (prop) => isPropValid(['fullHeight'], prop),
+  shouldForwardProp: (prop) =>
+    isPropValid(['fullHeight'], prop) && prop !== 'title',
```

Comment:
Is that the same logic as this?

```suggestion
    isPropValid(['fullHeight', 'title'], prop),
```

Reply:
I'd need to try but `title` is a valid prop so it could make reading this logic ambiguous

Reply:
I think the `isPropValid` need to be re-thought but I think it's intended to be used for this case

Reply:
yeah I looked it up and the name threw me off. It's got nothing to do with it being a valid prop 🤦 - will update

Also making a ticket to revisit this.

Reply:
> Also making a ticket to revisit isPropValid 

Thanks! Name threw me off too. 😅

Reply:
All done!